.github/workflows/test-environment.yml

```
@@ -0,0 +1,127 @@
+name: sanity workflow
```

Just for consistency 
```suggestion
name: Sanity tests
```